packages/components/box/docs.mdx

```diff
@@ -0,0 +1,86 @@
+---
+name: Box
+route: /components/box
+menu: Components
+---
+
+import { Playground, Props } from "docz"
+import Box from './index'
+import { Flex } from "@chakra-ui/core"
+
+# Box
+
+Box component
+
+The HTML DOM structure is
+
+```
+  <div></div>
+```
+
+
+
+## Usage
```

Comment:
Just linting MDX.

![image](https://user-images.githubusercontent.com/39694575/78244362-0f48c380-7503-11ea-84d9-5b556bc522af.png)

![image](https://user-images.githubusercontent.com/39694575/78244380-18d22b80-7503-11ea-89c3-fa8506769817.png)

Install the plugin and add the file association in vscode settings. You will get syntax highlighting, linter & prettier support

Reply:
will fix linting